src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.wrap {
+  position: relative;
+  margin-top: 20px;
+}
+
+.imgStyleBig {
+  background-image: url(images/Search.svg);
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.input {
+  box-sizing: border-box;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: rgba(61, 78, 97, 1);
+  width: 100%;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.inputBigStyle {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+  line-height: 22px;
+}
```

Comment:
checklist p2
move it to input, and than in small these styles will be overwritten